### PR TITLE
fix add rwsplit mode when  all  slave node is alive  that request to master node inner 725

### PR DESCRIPTION
**Pull request by ylinzhu**
Reason:  
  Improve 
Type:  
  Improve  
Influences：  
 fix add rwsplit mode when  all  slave node is alive  that request to master node inner 725

### CODE REVIEW

File: src/main/java/com/actiontech/dble/config/model/db/DbGroupConfig.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ public int getRwSplitMode() {
     }
 
     public void setRwSplitMode(int rwSplitMode) {
-        if (rwSplitMode >= 0 && rwSplitMode <= 2) {
+        if (rwSplitMode >= 0 && rwSplitMode <= 3) {
             this.rwSplitMode = rwSplitMode;
         } else {
             throw new ConfigException("dbGroup " + name + " rwSplitMode should be between 0 and 2!");
```
Comment (yanhuqing666) [inline src/main/java/com/actiontech/dble/config/model/db/DbGroupConfig.java:73]: between 0 and ?

Comment (ylinzhu) [reply src/main/java/com/actiontech/dble/config/model/db/DbGroupConfig.java]: between 0 and 3

**Comment by PanternBao** [comment]
may need to think of the scenario that user forces slave by hint.

**Comment by ylinzhu** [comment]
year, really not considered, has been modified

**Review by PanternBao — APPROVED** [review]
lgtm

**Pull request merged**